packages/fastify-vite/config.js

```diff
@@ -80,19 +80,20 @@ const DefaultConfig = {
     return (error, req, reply) => {
       if (config.dev) {
         console.error(error)
+        scope.vite.devServer.ssrFixStacktrace(error);
```

Comment:
in production it would end up crashing because it could not find a devServer under vite

Reply:
Should have not included the ; sorry